config/routes.rb

```diff
@@ -19,14 +19,3 @@
     root to: 'users#new'
   end
 
-Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  resources :posts do
-    resources :comments
-  end
-
-  get 'login', to: 'sessions#new'
-  post 'login', to: 'sessions#create'
-  get 'signup', to: 'users#new'
-  post 'signup', to: 'users#create'
-end
```

Comment:
I'm confused what's happened here - have you deleted everything from routes apart from `root to: 'users#new'` ? If so why, that seems like a mistake?

Reply:
I think you have only looked at the bottom of the file, if you view the whole file you can see there are some double routes. We just deleted the duplicates.

Reply:
AH yes! :D